src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+
+    public static final String VALID_EMAIL = "^[a-zA-Z0-9_!#$%&'*+/=?`{|}~^.-]+@[a-zA-Z0-9.-]+$";
```

Comment:
Let's create a constant of a type Pattern but not a String. Because otherwise there will be created a new object of this type each time you call the method `matches`